### DISCUSSION TITLE
Storing local temporary data

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17775] [Q&A]
What would be a good way to store temporary data in the pipeline (locally)?
We are using a docker compose setup, so the code runs in a separate container.
So if I store something in `/tmp` it won't be there on the next run.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699368937828329?thread_ts=1699368937.828329&cid=C01U954MEER).*

**Comment by jamiedemaria** [comment]
Can you create a volume and store your data there?

**Reply by Auric-Manteo** [reply]
Yes, that is definitely a possibility.
I was mostly wondering about best practices here for syncing data between different jobs, so I'm not missing a better solution.

**Reply by jamiedemaria** [reply]
Yeah I think a shared volume that both jobs know how to access (maybe using a resource if you want to be very dagster-y) is good for this case. I can't think of any other options that aren't more complicated
